tests/communities/conftest.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2022 CERN.
+# Copyright (C) 2022 Northwestern University.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Pytest configuration."""
+
+import itertools
+
+import pytest
+from flask_security import login_user
+from invenio_access.models import ActionRoles
+from invenio_access.permissions import superuser_access, system_identity
+from invenio_accounts.models import Role
+from invenio_accounts.testutils import create_test_user, login_user_via_session
+from invenio_admin.permissions import action_admin_access
+from invenio_records_resources.proxies import current_service_registry
+from invenio_vocabularies.contrib.affiliations.api import Affiliation
+from invenio_vocabularies.records.api import Vocabulary
+from invenio_vocabularies.records.models import VocabularyType
+
+from invenio_communities.communities.records.api import Community
+
+
+@pytest.fixture(scope="function")
+def superuser_role_need(db):
```

Comment:
some of this fixtures are already in the root confest? if so, no need to rewrite them here.